package.json

```diff
@@ -5,11 +5,10 @@
   "packageManager": "yarn@1.22.19",
   "main": "index.js",
   "scripts": {
-    "start": "webpack serve --config=webpack.config.dev.js ",
+    "start": "IPFS_DEPLOY=1 webpack serve --config=webpack.config.dev.js ",
```

Comment:
maybe IPFS_DEPLOY better be `bool`?

Reply:
no such type as bool in envs

Reply:
if I am not mistaken you can pass like: `MY_PARAM npm run ...` and it should be `true`
or `MY_PARAM=true`, maybe it will be string, but more clear than numbers, I think

Reply:
I don't have solid knowledge in scrips, but for reading I don't like 0 1 (then someone adds '2' mode...).
It is not critical, other devs can say what they thinking